models/articles.model.js

```diff
@@ -22,15 +40,24 @@ exports.fetchArticleById = async (article_id) => {
   `,
     [article_id]
   );
-  console.log(res.rows, "<-- res.rows in model");
   if (res.rows.length === 0) {
     return Promise.reject({ status: 404, msg: "Invalid article_id" });
   } else {
     return res.rows[0];
   }
 };
 
-exports.updateArticle = async (body, article_id) => {
+exports.fetchCommentsByArticleId = async (article_id) => {
+  const res = await db.query(
```

Comment:
Spot on code... but how about deconstructing rows here to save you a few characters on line 57? 😄

Reply:
Think (?) I follow... like this?

 const { rows } = await db.query(
    `
  SELECT * FROM comments
  WHERE article_id = $1`,
    [article_id]
  );
  return rows;

Reply:
That looks good to me!